packages/perseus-editor/src/widgets/input-number-editor.jsx

```diff
@@ -45,22 +46,44 @@ const answerTypes = {
     },
 };
 
-type Props = $FlowFixMe;
+type Props = {|
+    value: number,
+    simplify: React.ElementConfig<typeof InputNumber.widget>["simplify"],
+    size: React.ElementConfig<typeof InputNumber.widget>["size"],
+    inexact: React.ElementConfig<
+        typeof InputNumber.widget,
+    >["reviewModeRubric"]["inexact"],
+    maxError: React.ElementConfig<
+        typeof InputNumber.widget,
+    >["reviewModeRubric"]["maxError"],
+    answerType: React.ElementConfig<typeof InputNumber.widget>["answerType"],
+    rightAlign: React.ElementConfig<typeof InputNumber.widget>["rightAlign"],
 
-class InputNumberEditor extends React.Component<Props> {
-    static propTypes = {
-        value: PropTypes.number,
-        simplify: PropTypes.oneOf(["required", "optional", "enforced"]),
-        size: PropTypes.oneOf(["normal", "small"]),
-        inexact: PropTypes.bool,
-        maxError: PropTypes.number,
-        answerType: PropTypes.string,
-        rightAlign: PropTypes.bool,
-    };
+    onChange: ({|
+        value?: ParsedValue | 0,
```

Comment:
This is a little weird, but I imagine this is just weird existing logic? It can be an object in the shape of `{ value: Number, exact: Boolean }` or `0`?

Reply:
It _is_ weird... and I think a bug. But for now I wanted to just match the Flow types to what the code was doing instead of going down a rabbit hole to also fix this editor. My suspicion is that because this is a legacy widget, nobody has edited content with this widget in a while and so we haven't hit this bug yet.